### ISSUE TITLE
Project Creation Failed - [ErrNo 13] Permission denied u'c:\\oauth2service.json'

### ISSUE THREAD

**Issue by daveybo** [jay0lee/GAM#782]
The issue tracker is for reporting product deficiencies. "How do I?" questions should be posted to the discussion forum at https://groups.google.com/group/google-apps-manager. When in doubt, start at the discussion forum and return here only when instructed to do so.

Please confirm the following:
* I have upgraded to the latest GAM release from https://git.io/gamreleases and I still have this issue.
* I am typing the command as described in the GAM Wiki at https://github.com/jay0lee/gam/wiki

Full steps to reproduce the issue:
1. ran MSI 64 bit installer for latest release
2. entered admin email address
3. last message in cmd window was "creating service account"

Expected outcome (what are you trying to do?):
setup GAM to add classroom courses in bluk from a CSV file
Actual outcome (what errors or bad behavior do you see instead?):

Project Creation Failed - [ErrNo 13] Permission denied u'c:\\oauth2service.json'

**Comment by belarminobrunoz** [comment]
Anyone knows how to solve this error?

I had this same problem today.

The error message was

`ERROR: {'name': 'operations/cp.8844208477150365015', 'metadata': {'@type': 'type.googleapis.com/google.cloudresourcemanager.v1.ProjectCreationStatus', 'ready': True}, 'done': True, 'error': {'code': 7, 'message': "Permission 'resourcemanager.projects.create' denied on parent resource 'organizations/5555555555555'.", 'details': [{'@type': 'type.googleapis.com/google.rpc.ResourceInfo', 'resourceType': 'project', 'resourceName': 'projects/gam-project-s60-4r4-wdj'}]}}`

**Comment by jay0lee** [comment]
This indicates the user you authorized as does not have permissions to create GCP projects in your organization. Do you know who does?

**Comment by m-o-n-t-y** [comment]
Having this issue today.

ERROR: [Errno 13] Permission denied: 'C:\\Program Files\\GYB\\oauth2service.json'

It's a personal Gmail account, so I should have all rights(?). I did navigate into one of the GCP projects that was created and and clicked "Enable" on the Secret Manager API and got the message "Billing required". Do I need a paid GCP account? I searched the GYB docs and didn't see a mention of that. Thanks for your help.

![image](https://user-images.githubusercontent.com/32070390/98286639-4b994200-1f72-11eb-9733-070bc7b39c23.png)
![image](https://user-images.githubusercontent.com/32070390/98287024-c82c2080-1f72-11eb-8c32-56a133430372.png)
